core/types/banking.go

```diff
@@ -140,13 +140,13 @@ func GovernanceTransferFromProto(g *checkpointpb.GovernanceTransfer) *Governance
 
 func (g *GovernanceTransfer) IntoEvent(amount *num.Uint, reason *string) *eventspb.Transfer {
 	// Not sure if this symbology gonna work for datanode
-	from := "0000000000000000000000000000000000000000000000000000000000000000"
+	from := "*"
```

Comment:
Is that change expected?

Reply:
doesn't seem to belong here, probably sneaked in in some merge